qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -13,8 +13,9 @@
         "heading": "Accessing  NIAID  Clinical Trials Data",
         "multiLineTexts": [
           "AccessClinicalData@NIAID is a NIAID cloud-based, secure data platform that enables sharing of and access to data sets from NIAID COVID-19 and other sponsored clinical trials for the basic and clinical research community.",
-          "AccessClinicalData@NIAID can help support your Data Management Plan (DMP) to share clinical research data in compliance with the 2023 NIH Data Management and Sharing Policy.",
-	        "<p style='margin-left: 20px; font-style: italic;'>De-identified human clinical research and trials data can be submitted to <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>accessclinicaldata@NIAID</a>, which is a NIAID cloud-based, controlled access, secure data platform that enables sharing of and access to human clinical research and trials data sets and related metadata for the basic and clinical research community.  There is no cost to share and access data in this NIAID supported data platform.</p>",
+          "AccessClinicalData@NIAID can help support your Data Management Plan (DMP) to share clinical research data in compliance with the NIH 2023 Data Management and Sharing Policy.",
+	        "<p style='margin-left: 20px; font-style: italic;'>De-identified human clinical research and trials data can be submitted to accessclinicaldata@NIAID, which is a NIAID cloud-based, controlled access, secure data platform that enables sharing of and access to human clinical research and trials data sets and related metadata for the basic and clinical research community.  There is no cost to share and access data in this NIAID supported data platform.</p><br />",
+          "<p style='margin-left: 20px; font-style: italic;'>If you are interested in making your data available, please contact us at <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>accessclinicaldatasupport@niaid.nih.gov</a>.</p>",
```

Comment:
I think Maria wanted the hyper link to be on "contact us". 
So: 
`please <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>contact us</a>.`